Update error-codes.md (#1089)
MicrosoftDocs/dataexplorer-docs: data-explorer/error-codes.md

| BadRequest_IngestionPropertyNotSupportedInThisContext | Ingestion property isn't supported in this context.| Permanent |
| BadRequest_InvalidBlobUri                        | Blob URI is invalid.      | Permanent          |
| BadRequest_DataCapacityLimitReached              | Storage account has reached its data capacity limit.      | Permanent          |

> | BadRequest_InvalidManagedIdentity                | Managed Identity either doesn't exist or doesn't has the right permissions.      | Permanent          |

Reviewer: ```suggestion
| BadRequest_InvalidManagedIdentity                | Managed Identity either doesn't exist or doesn't have the required permissions.      | Permanent          |
```
